ui/raidboss/data/04-sb/raid/o8n.js

```diff
@@ -67,16 +58,7 @@
       regexJa: Regexes.startsUsing({ id: '2929', source: '神々の像', capture: false }),
       regexCn: Regexes.startsUsing({ id: '2929', source: '众神之像', capture: false }),
       regexKo: Regexes.startsUsing({ id: '2929', source: '신들의 상', capture: false }),
-      alertText: {
-        en: 'Get Right/East =>',
-        fr: 'Allez à Droite/Est =>',
-        de: 'Nach Rechts/Westen =>',
-      },
-      tts: {
-        en: 'right',
-        fr: 'Projection depuis le côté droit',
-        de: 'rechts',
-      },
+      response: Responses.goRight(),
```

Comment:
I think right/east => is important here, as there's not a clear left/right.  Same for all the other o8 ones.

Reply:
fixed by changing it to goEast (for Right) and goWest(for Left) as they return always Left/West or Right/East